playbooks/ran/configure-lvm-storage.yml

```diff
@@ -0,0 +1,135 @@
+---
+# This playbook configures LVM storage on OpenShift master nodes and creates
+# LocalVolume resources for local storage operator.
+#
+# Prerequisites:
+#   - SSH access to master nodes
+#   - Valid kubeconfig for cluster authentication
+#   - Local Storage Operator must be installed
+#
+# Usage:
+#   ansible-playbook -i inventories/ocp-deployment/build-inventory.py \
+#     playbooks/configure-lvm-storage.yml \
+#     --extra-vars "kubeconfig=/path/to/kubeconfig"
+#
+# What it does:
+#   1. Creates LVM volume group and logical volumes on master nodes
+#   2. Creates OpenShift LocalVolume CRs to expose the volumes to the cluster
+#   3. Sets local-sc as the default storage class
+
+- name: Configure LVM storage on master nodes
+  hosts: masters
+  become: true
+  remote_user: core
+  gather_facts: false
+
+  tasks:
+    - name: Create volume group new_vol_group on /dev/vda
+      community.general.lvg:
+        vg: new_vol_group
+        pvs: /dev/vda
+        state: present
```

Comment:
maybe we should drop the current table before creating a new one.

cause in case of a pre configured host this will either save the current or fail